lib/whitehall/carrier_wave/sanitized_file.rb

```diff
@@ -1,48 +1,7 @@
 module CarrierWave
   class SanitizedFile
-    # CarrierWave has been designed with the assumption that your files will
```

Comment:
Reading this comment, I'm not sure I did the right thing in "simplifying" this code.  Will have to try it out on integration.